features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/wrappers/BitmapWrapper.kt

```diff
@@ -55,6 +56,16 @@ internal class BitmapWrapper(
                 e
             )
             null
+        } catch (e: RuntimeException) {
+            // It's still possible that RuntimeException is thrown after checking the bitmap
+            // is not recycled.
+            logger.log(
+                level = InternalLogger.Level.ERROR,
+                target = InternalLogger.Target.USER,
```

Comment:
is it useful to send it to the telemetry at least to know how often it happens? not sure if it is actually actionable though, unless we change the way we are processing bitmaps.

Reply:
It's interesting to know the data, but I can hardly imagine the action to take even if we see the data is up or down. Because we are kind of sure that it's caused by GC, and we can do nothing about it.